.github/dependabot.yml

```
@@ -0,0 +1,21 @@
+enable-beta-ecosystems: true
+updates:
+ - package-ecosystem: "devcontainers"
+   directory: "/"
+   schedule:
+     interval: weekly
+ - package-ecosystem: "docker"
+   directory: "/"
+   schedule:
+     interval: weekly
+ - package-ecosystem: "uv"
+   directory: "/" # Location of package manifests
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)